aws-mssql.yml

```diff
@@ -11,8 +11,33 @@ support_url: https://aws.amazon.com/sql/
 tags: [aws, mssql, beta]
 plan_updateable: true
 provision:
-  plan_inputs: [] # Expected: mssql_version, storage_gb
+  plan_inputs: []
   user_inputs:
+  - field_name: engine
+    required: true
+    type: string
+    details: The edition for the MSSQL instance.
+    default: sqlserver-ee
+    enum:
+      sqlserver-ee: sqlserver-ee
```

Comment:
The use of enums is discourage unless it is a field we don't expect to change much. If this is the case, then enums is ok.

Reply:
In this case the usage of enum is justified since:
- We can't leave the field without validation. Otherwise customers would be able to specify "mysql", "postgres", "oracle" here which could cause many unpredictable issues.
- The list of enumerated values it is not expected to change.